test/e2e/storage/drivers/csi.go

```diff
@@ -461,6 +461,9 @@ func (g *gcePDExternalCSIDriver) SkipUnsupportedTest(pattern testpatterns.TestPa
 	if pattern.FsType == "xfs" {
 		framework.SkipUnlessNodeOSDistroIs("ubuntu", "custom")
 	}
+	if pattern.FeatureTag == "sig-windows" {
+		framework.SkipUnlessNodeOSDistroIs("windows")
```

Comment:
I think we want to skip entirely if it's windows since the CSI driver doesn't support it?  Also, it's a little confusing, but there are two gcepd CSI drivers defined here, so this should go in both.  (I eventually want to remove both pd csi driver definitions, tracked in https://github.com/kubernetes/kubernetes/issues/70258)

Reply:
got it. fixed. Thanks!